### PR TITLE
[ZT] ZTWA learning path

### PR DESCRIPTION
**Pull request by maxvp**

### CODE REVIEW

File: content/cloudflare-one/_partials/access/_isolation-prereqs.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+_build:
+  publishResources: false
+  render: never
+  list: never
+---
+
+Your browser must [allow third-party cookies](/cloudflare-one/identity/authorization-cookie/#allow-third-party-cookies-in-the-browser) on the application domain.
```
Comment (maxvp) [inline content/cloudflare-one/_partials/access/_isolation-prereqs.md:8]:
```suggestion
<div class="special-class" markdown="1">

Your browser must [allow third-party cookies](/cloudflare-one/identity/authorization-cookie/#allow-third-party-cookies-in-the-browser) on the application domain.

</div>
```

File: content/learning-paths/zero-trust-web-access/migrate-applications/best-practices.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Best practices
+pcx_content_type: overview
+weight: 3
+layout: learning-unit
+---
+
+Most customers have a heterogenous private application portfolio; some are home-built, some are internal managed services, some have SSO integrations available, and some rely on HTML or other forms of authentication. With that in mind, we recommend that you mix-and-match [onboarding solutions](/learning-paths/zero-trust-web-access/migrate-applications/integrated-sso/#potential-solutions) to fit the needs of each individual application. As shown in the table below, you can bucket applications into a series of stack-ranked categories that prioritize ease of implementation and total organizational impact.
+
+| Application type | Recommendation | Time investment | Outcome |
```
Comment (maxvp) [inline content/learning-paths/zero-trust-web-access/migrate-applications/best-practices.md]: Is the `Time investment` column necessary if all entries except one are low?

File: content/learning-paths/zero-trust-web-access/advanced-workflows/isolate-application.md [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+- **Policy 2: Block credit card numbers**
+
+{{<tabs labels="Dashboard | API">}}
+{{<tab label="dashboard" no-code="true">}}
+
+| Selector                     | Operator | Value                      | Logic | Action  |
+|------------------------------|----------|----------------------------|-------|---------|
+| Host                         | in       | `internal.site.com`        | And   | Block   |
+| [DLP Profile](/cloudflare-one/policies/data-loss-prevention/dlp-profiles/)                  | in       | `Financial information`    | And   |         |
+| Passed Device Posture Checks | not in   | `Corporate serial numbers` |       |         |
```
Comment (maxvp) [inline content/learning-paths/zero-trust-web-access/advanced-workflows/isolate-application.md]:
```suggestion
| [DLP Profile](/cloudflare-one/policies/data-loss-prevention/dlp-profiles/)                  | in       | _Financial Information_    | And   |         |
| Passed Device Posture Checks | not in   | _Corporate serial numbers_ |       |         |
```

File: content/learning-paths/zero-trust-web-access/customize-ux/tags.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+title: Tags
+pcx_content_type: overview
+weight: 2
+layout: learning-unit
+---
+
+{{<render file="access/_tags.md" productFolder="cloudflare-one">}}
```
Comment (maxvp) [inline content/learning-paths/zero-trust-web-access/customize-ux/tags.md:8]: This makes the H3 tags from the partial respect page structure:
```suggestion
## Manage tags

{{<render file="access/_tags.md" productFolder="cloudflare-one">}}
```

File: content/learning-paths/zero-trust-web-access/terraform/publish-apps-with-terraform.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+This guide covers how to use the [Cloudflare Terraform provider](https://registry.terraform.io/providers/cloudflare/cloudflare/latest/docs) to quickly publish and secure a private application. In the following example, we will add a new public hostname route to an existing Cloudflare Tunnel, configure how `cloudflared` proxies traffic to the application, and secure the application with Cloudflare Access.
+
+## Prerequisities
+
+- [Domain on Cloudflare](/learning-paths/zero-trust-web-access/initial-setup/add-site/)
+- [IdP integration](/learning-paths/zero-trust-web-access/initial-setup/configure-idp/)
+- [Cloudflare Tunnel](/learning-paths/zero-trust-web-access/connect-private-applications/create-tunnel/#create-a-tunnel) created via the Zero Trust dashboard
+- [Terraform client](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
+- [API token](https://developers.cloudflare.com/fundamentals/api/get-started/create-token/) (refer to the [minimum required permissions](/cloudflare-one/connections/connect-networks/deploy-tunnels/deployment-guides/terraform/#3-create-a-cloudflare-api-token))
```
Comment (maxvp) [inline content/learning-paths/zero-trust-web-access/terraform/publish-apps-with-terraform.md]:
```suggestion
- [Add your domain to Cloudflare](/learning-paths/zero-trust-web-access/initial-setup/add-site/)
- [Configure an IdP integration](/learning-paths/zero-trust-web-access/initial-setup/configure-idp/)
- [Create a Cloudflare Tunnel](/learning-paths/zero-trust-web-access/connect-private-applications/create-tunnel/#create-a-tunnel) via the Zero Trust dashboard
- Install the [Terraform client](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
- [Create an API token](https://developers.cloudflare.com/fundamentals/api/get-started/create-token/) (refer to the [minimum required permissions](/cloudflare-one/connections/connect-networks/deploy-tunnels/deployment-guides/terraform/#3-create-a-cloudflare-api-token))
```

File: content/learning-paths/zero-trust-web-access/migrate-applications/best-practices.md (hunk 2 above)
Comment (ranbel) [reply content/learning-paths/zero-trust-web-access/migrate-applications/best-practices.md]: Good point. I will remove it.

**Review by ranbel — APPROVED** [review]

**Pull request merged**